src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { BsTwitter,BsInstagram , BsGithub } from 'react-icons/bs';
+
+function Footer(){
+
+    return<div>
+
+        <div className='home-footer flex-center flex-col'>
+            <div className='social-link padd-top-sm'>
+                <a href='/'><div className='avatar-sm avatar-round flex-center footer-avatar' ><BsTwitter/></div> </a>
```

Comment:
Use <Link/> tag wherever there is< a> tag

Reply:
yeah , i will take care this thank you pointing out